SwrveSDK/SDK/Conversation/SwrveConversationCampaign.h

```diff
@@ -10,13 +10,14 @@
 @property (nonatomic, retain) NSArray* filters;                     /*!< Filters needed to display this campaign. */
 
 /*! Check if the campaign has any conversation setup for the
- * given trigger.
+ * given trigger and parameters associated
  *
  * \param event Trigger event.
+ * \param paramters Dictionary of parameters associated (nullable)
  * \returns TRUE if the campaign contains a conversation for the
  * given trigger.
  */
--(BOOL)hasConversationForEvent:(NSString*)event;
```

Comment:
I'm keeping the old one, but all it does it call HasConversationForEvent with nil as parameters